apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.BenchmarkingTool/Program.cs

```diff
@@ -0,0 +1,244 @@
+// Copyright 2022 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Download;
+using Google.Apis.Storage.v1.Data;
+using Google.Apis.Upload;
+using Google.Cloud.ClientTesting;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+
+namespace Google.Cloud.Storage.V1.BenchmarkingTool
+{
+    public static class Program
+    {
+        private static readonly string s_projectId = TestEnvironment.GetTestProjectId(TestEnvironment.TestProjectEnvironmentVariable);
+
+        private static readonly string[] s_logHeaders = new string[] { "Op", "ObjectSize", "AppBufferSize", "LibBufferSize", "Crc32cEnabled", "MD5Enabled", "ApiName", "CpuTimeUs", "ElapsedTimeUs", "Status" };
+
+        private static Stopwatch s_stopwatch;
+
+        private const long TicksPerMicrosecond = 10;
+
+        private static void Main(string[] args)
+        {
+            try
+            {
+                var configuration = Configuration.FromCommandLineArguments(args);
+
+                var logger = new PerformanceLogger(configuration.OutputFolder);
+                logger.Log(s_logHeaders);
+
+                var client = StorageClient.Create();
+
+                s_stopwatch = Stopwatch.StartNew();
+
+                for (int i = 1; i <= configuration.NumOfSamples; ++i)
```

Comment:
I tend to think that `i++` is more idiomatic for loop iterations. (Obviously it doesn't matter which is used here, but it's good to be consistent across our codebase.)